packages/forms/src/styled/input-group/StyledInputGroup.ts

```diff
@@ -79,6 +80,10 @@ const itemStyles = (props: ThemeProps<DefaultTheme>) => {
       z-index: -1;
     }
 
+    & > ${StyledTextInput}:first-child {
+      margin-${horizontal}: 0 !important; /* [2] */
+    }
+
```

Comment:
Is this entire block necessary, or does `!important` just need to be added to the `& > *:first-child:not(:last-child)` block below?